### PR TITLE
Add Langrisser Re:Incarnation Tensei

### PR DESCRIPTION
**Pull request by dmiltso**
Adds Langrisser Re:Incarnation Tensei.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/langrisser-re-incarnation-tensei/game.dat [hunk 1]
```diff
@@ -0,0 +1,25 @@
+title = "Langrisser Re:Incarnation Tensei"
+description = "Developer: Masaya Games\nTactical: role-playing game\nThe game takes place in a flooded world and the story's protagonist is a man with the Sacred Sword Langrisser, looking to reunite himself with a childhood friend, while fighting the empire's army. The battle system is the same as known from other Langrisser titles, but with 10-on-10 battles fought by super-deformed 3D characters serving as units. The units consist of Infantry, Cavalry, and Archers."
```
Comment (Hexagon12) [inline games/langrisser-re-incarnation-tensei/game.dat]: Line breaks (\n) should be removed.

File: games/langrisser-re-incarnation-tensei/game.dat [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+release_date = "2015-07-23"
+
+[[testcases]]
+title = "0004000000187F00"
+compatibility = "3"
+date = "2017-09-05"
+version = "HEAD-0ceaf22"
+cpu = "Intel Xeon E5450"
+gpu = "GeForce GTX 550 Ti"
+os = "Windows 7 617601"
```
Comment (Hexagon12) [inline games/langrisser-re-incarnation-tensei/game.dat]: _Windows 7 7601_ should be fine.

File: games/langrisser-re-incarnation-tensei/game.dat [hunk 3]
```diff
@@ -0,0 +1,25 @@
+title = "Langrisser Re: Incarnation Tensei"
+description = "Developer: Masaya Games. Tactical: Role-Playing game. The game takes place in a flooded world and the story's protagonist is a man with the Sacred Sword Langrisser, looking to reunite himself with a childhood friend, while fighting the empire's army. The battle system is the same as known from other Langrisser titles, but with 10-on-10 battles fought by super-deformed 3D characters serving as units. The units consist of Infantry, Cavalry, and Archers."
```
Comment (Leo626) [inline games/langrisser-re-incarnation-tensei/game.dat]: You can leave out the sentence about the plot. I would prefer the description to start with:
```Langrisser Re: Incarnation Tensei is a Tactical role-playing game developed by Masaya Games. It was published in Japan by Extreme Games, and in North America by Aksys Games.```
and just add the last two sentence about the game play.

File: games/langrisser-re-incarnation-tensei/game.dat [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+needs_system_files = false
+needs_shared_font = false
+game_type = "eshop"
```
Comment (Leo626) [inline games/langrisser-re-incarnation-tensei/game.dat]: This should be `''3ds''` as the game had a retail release.

File: games/langrisser-re-incarnation-tensei/game.dat [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+[[releases]]
+title = "000400000015E700"
+region = "jpn"
+release_date = "2015-07-23"
+
+[[testcases]]
+title = "0004000000187F00"
+compatibility = "3"
+date = "2017-09-05"
+version = "HEAD-0ceaf22"
```
Comment (Leo626) [inline games/langrisser-re-incarnation-tensei/game.dat]: This doesn't seem to be tested with Nightly.
```
C:\projects\citra>git show 0ceaf22
fatal: ambiguous argument '0ceaf22': unknown revision or path not in the working tree.
```

**Comment by Leo626** [comment]
Since your screenshots are in English, I would prefer the boxart to be the North American one.

**Comment by Leo626** [comment]
LGTM

**Pull request merged**